scripts/docker-run-abi-test.sh

```diff
@@ -34,7 +34,7 @@ create_base_container() {
   # Run a Postgres container
   docker run -u postgres -d --name $1 -v ${BASE_DIR}:/src -v ${COMPILE_VOLUME}:/compile $2
   # Install build dependencies
-  docker exec -u root $1 /bin/bash -c "apk add --no-cache --virtual .build-deps coreutils dpkg-dev gcc libc-dev make util-linux-dev diffutils cmake bison flex openssl-dev && mkdir -p /build/debug"
+  docker exec -u root $1 /bin/bash -c "apk add --no-cache --virtual .build-deps coreutils dpkg-dev gcc libc-dev make util-linux-dev diffutils cmake bison flex openssl-dev git && mkdir -p /build/debug"
```

Comment:
I think there was a point to previously not having git in all our builds. It ensures that all tests pass irrespective of having this binary installed. And, why do we need Git installed for this build if we anyway don't build in a Git repo? 

Thus, I suspect that this change isn't needed.

Reply:
Yes, I do not think it is needed any more now that we are not running the test if Git is not installed.